app/views/ProjectEdit/GeoAreas/RegionCard/index.tsx

```diff
@@ -497,20 +567,36 @@ function RegionCard(props: Props) {
             onExpansionChange={handleExpansion}
             expansionTriggerArea="arrow"
             headerActions={(
-                <QuickActionConfirmButton
-                    name="deleteButton"
-                    title="Remove geo area from this project"
-                    onConfirm={handleDeleteRegionClick}
-                    message="Removing the geo area will remove all the tagged geo data under your project.
-                    The removal of tags cannot be undone.
-                    Are you sure you want to remove this geo area from the project?"
-                    showConfirmationInitially={false}
-                    disabled={navigationDisabled}
-                >
-                    <IoTrashBinOutline />
-                </QuickActionConfirmButton>
+                <>
+                    {region.status === 'FAILED' && (
```

Comment:
Check if the region is already published. If the region is already published this check is not necessary as the user won't/shouldn't be able to retrigger.